hack/templates/values/values.template

```diff
@@ -17,37 +17,51 @@
 # Declare variables to be passed into your templates.
 
 nfd:
+  # -- Deploy Node Feature Discovery operator.
   enabled: true
+  # -- Deploy Node Feature Rules to label the nodes with the discovered features.
   deployNodeFeatureRules: true
 
+# -- Enable CRDs upgrade with helm pre-install and pre-upgrade hooks.
 upgradeCRDs: true
 
 sriovNetworkOperator:
+  # -- Deploy SR-IOV Network Operator.
   enabled: false
 
-# Node Feature discovery chart related values
+# Set both enableNodeFeatureApi and NodeFeatureAPI feature gate to false to disable.
 node-feature-discovery:
-  # set both enableNodeFeatureApi and NodeFeatureAPI feature gate to false to disable
+  # -- The Node Feature API is used to expose node-specific hardware and 
```

Comment:
this is incorrect IMO. that just an internal mechanism how NFD works.

when node feature API is enabled communication between nfd master and worker is done through NodeFeature CRs
otherwise through gRPC.

NFD will work the same when this is enabled or disabled. (in the future grpc commnication mode will be removed)

Reply:
Thanks for explaining this. I will update it.